### ISSUE TITLE
Complete support for HuggingFace Formats

### ISSUE THREAD

**Issue by madpah** [sonatype-nexus-community/sonatype-platform-browser-extension#204]
Complete support for as many formats on Hugging Face as is practical.

See: https://help.sonatype.com/en/hugging-face-model-analysis.html

| Format | Supported ? | Example |
| ------- | ------------ | --------- |
| Pytorch (.bin) | ✅ | https://huggingface.co/distilbert/distilbert-base-uncased/tree/main | 
| Pytorch (.pt) | ✅ via #205 | | 
| Pytorch (.pth) | ✅ via #205 | https://huggingface.co/kaliansh/sdrep/tree/main | 
| Pytorch (.pkl) | ✅ via #205 | https://huggingface.co/ymarin/pickle/tree/main | 
| Pytorch (.pickle) | ✅ via #205 | | 
| Safetensors (.safetensors) | ✅ | https://huggingface.co/distilbert/distilbert-base-uncased/tree/main | 
| Tensorflow (.h5) | ✅ | https://huggingface.co/distilbert/distilbert-base-uncased/tree/main | 
| Tensorflow.js (.bin) | ❌ | See #209  | 
| OpenVino (.bin) | ✅ via #205 | https://huggingface.co/OpenVINO/Phi-3.5-vision-instruct-int8-ov/tree/main | 
| MLC-LLM (.bin) | ✅ via #205 | https://huggingface.co/mlc-ai/TinyLlama-1.1B-Chat-v0.4-q0f16-MLC/tree/main | 
| GGUF (.gguf) | ✅ | https://huggingface.co/OuteAI/OuteTTS-1.0-0.6B-GGUF/tree/main | 
| Rust (.ot) | ✅ via #205 | https://huggingface.co/distilbert/distilbert-base-uncased/tree/main | 
| Transformers Pytorch (.bin.index.json) | ✅ via #205 | https://huggingface.co/mosaicml/mpt-7b/tree/main | 
| Transformers Safetensors (.safetensors.index.json) | ✅ via #205 | https://huggingface.co/microsoft/phi-2/tree/main | 
| Transformers TensorFlow (.h5.index.json) | ❌ | See #210 | 
| ONNX (.onnx) | ✅ via #205 | https://huggingface.co/optimum/bert-base-uncased-for-masked-lm/tree/main | 
| GGML (.bin) | ❌ | See #208 | 
| Transformers Flax (.msgpack.index.json) | ❌ | See #211 | 
| Flax/Jax (.msgpack) | ✅ via #205 | https://huggingface.co/distilbert/distilbert-base-uncased/tree/main | 
| FastText (.bin) | ❌ | See #212 https://huggingface.co/elishowk/fasttext_test2/tree/main |

**Comment by bryanwhyte** [comment]
The ICON summary count doesn't' match the Policy Violation count for https://huggingface.co/distilbert/distilbert-base-uncased/tree/main
<img width="1745" height="558" alt="Image" src="https://github.com/user-attachments/assets/38d823be-91ed-48f0-ae48-c30ae7abbbd9" />
